.github/workflows/lint.yml

```diff
@@ -14,12 +14,19 @@
 
 name: Lint
 
-on: [pull_request]
+on:
+  pull_request:
+    branches:
+    - main
```

Comment:
We do use pull requests against other branches, in which case we want this to run